app/views/shared/_maps_banner.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="l-maps_banner<%= ' l-maps_banner--non-sticky' if hide_sticky_header? %><%= ' l-maps_banner--home' if show_home_banner? %>">
+  <div class="l-constrained">
+    <div class="l-maps_banner__content">
+      <a href="https://moneyandpensionsservice.org.uk/" target="_blank" class="maps_banner__link">
+        <span class="maps_banner__mps-text">
+          <span><%= t('maps_banner.mps_provided_1') %></span>
+          <span><%= t('maps_banner.mps_provided_2') %></span>
+        </span>
+
+        <span class="maps_banner__mps-logo"></span>
+
+        <span class="visually-hidden">opens in a new window</span>
```

Comment:
Why exactly is this span needed?

Reply:
For accessibility. Most of our links to external sites have this, so it can be read by screen-readers.